crates/lakekeeper/src/catalog/namespace.rs

```diff
@@ -72,15 +73,31 @@ impl<C: Catalog, A: Authorizer + Clone, S: SecretStore>
             .await?;
         let mut t = C::Transaction::begin_read(state.v1_state.catalog.clone()).await?;
 
+        let mut can_list_everything = false;
```

Comment:
If we also add the Warehouse `can_list_everything` in OpenFGA, this is where we could use it to initialize.